dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -585,6 +586,21 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 					</StandFirstGrid>
 				</ElementContainer>
 
+				<ElementContainer
```

Comment:
I may be wrong, was just curious if we also need to use the switch in there too?

Reply:
You're right! thanks Marjan!